requirements.txt

```
@@ -1,4 +1,4 @@
-inflection==0.3.1
-coreapi==2.3.3
-hal_codec==1.0.2
-requests==2.22.0
+inflection>=0.3.1
+coreapi>=2.3.3
+hal_codec>=1.0.2
+requests>=2.22.0
```

Hi, thanks for your comments, I am happy to integrate this, but I'd rather stay with "compatible" version. What do you think? 
```suggestion
inflection~=0.3.1
coreapi~=2.3.3
hal_codec~=1.0.2
requests~=2.22.0
```

---

If I understand the [syntax](https://peps.python.org/pep-0440/#compatible-release) correctly `requests~=2.22.0` in this case would allow to go to a version 2.22.1 and all higher 'micro' versions (because it is compatible with `requests >= 2.22.0 , requests ==2.22.*` ), but would not allow installing 2.28.0 (which is what is required by another package within my conda env). 

In order to achieve compatibility for my particular case, the requirements would need to be relaxed to something like this:

```suggestion
inflection~=0.3.1
coreapi~=2.3.3
hal_codec~=1.0.2
requests~=2.22
```

---

I would say that's ok with this. Let me do a bit of testing before I merge this and publish a new release then.